src/OrchardCore/OrchardCore.Application.Cms.Targets/ServiceExtensions.cs

```diff
@@ -16,7 +16,11 @@ public static IServiceCollection AddOrchardCms(this IServiceCollection services)
                 .AddAuthentication()
                 .AddDataProtection()
 
-                .AddSetupFeatures("OrchardCore.Setup")
+                .AddSetupFeatures(
+					"OrchardCore.Setup",
+					"OrchardCore.Apis.GraphQL",
```

Comment:
Nope. Recipe only, only if we want it by default, which I don't think so for the recipes we have today.

Reply:
In recipes, yes (again if we had such a recipe), not hard coded in the host application.

Reply:
Shifting to the Setup module